packages/rich-text-editor/src/RichTextEditor/components/Toolbar/Toolbar.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { Children, useState, useContext, useRef, ReactNode } from "react"
+import classnames from "classnames"
+import { ToolbarSectionProps } from "../ToolbarSection"
+import { ToolbarItem, ToolbarItemProps } from "../ToolbarItem"
+import styles from "./Toolbar.scss"
+
+export interface ToolbarProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
We should do the `classNameOverride` thing here, and we can use the `@kaizen/component-base` stuff that Cass built. E.g. https://github.com/cultureamp/kaizen-design-system/pull/2632/files#diff-6da5f01f257e37406d113e4bf53b6547954456add88abfd0b8254790fda90515R3